src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Mappers/DomainToDal.cs

```diff
@@ -9,13 +9,15 @@
     using Domain.TeamsAggregate;
     using Domain.TournamentRequestAggregate;
     using Domain.TournamentsAggregate;
-    using VolleyManagement.Domain.UsersAggregate;
+    using Domain.UsersAggregate;
     using Entities;
 
+#pragma warning disable S1200 // Classes should not be coupled to too many other classes (Single Responsibility Principle)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)